### ISSUE TITLE
Add "Docs: Juju developers" to Learn menu

### ISSUE THREAD

**Issue by tmihoc** [canonical-web-and-design/juju.is#428]
Juju currently has 2 official sets of docs, OLM and SDK. I'd like to add a third, "Developers". This category is already there on juju.is/docs (it's the third item in the side navigation). However, it needs to be added to the Learn menu (top right, under OLM and SDK). Also, it needs to be given its own page etc. so as to behave the same as the OLM and SDK docs. @bartaz was telling me I can make that happen by filing an issue here. Happy to clarify further any time. PS Could we make this a priority? Thanks!

**Comment by carkod** [comment]
Hi @tmihoc but where is the content of this "Juju developers" docs?

As an example, if you click on OLM documentation and go down to the "[Help improve this document in the forum](https://discourse.charmhub.io/t/juju-olm-documentation/1087)." and click on that, you will be redirected to the content of the docs which is hosted on discourse. You'd need the same for juju developers, otherwise the site doesn't have anything to show?

**Comment by tmihoc** [comment]
@carkod Hi Carlos, I'm planning to link a bunch of Discourse docs in the Navigation of that page. I already have a list ready but am waiting for this page to become functional before I list them. Once it's up, I'll be able to link things there within minutes.

**Comment by tmihoc** [comment]
@anthonydillon I have a bunch of docs that I want to add to it but won't do it now as if I link them in the Navigation now, my guess is they will be displayed incorrectly on juju.is/docs instead of juju.is.docs/dev . I am not aware of any broken links. In short, yes, I am sure.

**Comment by anthonydillon** [comment]
The link I checked was "Docs for juju developers" on https://juju.is/docs/dev.

**Comment by tmihoc** [comment]
@anthonydillon Yes, that's the page I want linked in Learn. Once it's linked, I'll add docs to it right away as I already have a list ready. As I was saying, the only reason I'm waiting is because if I add them now my guess is that they will incorrectly show on juju.is/docs instead of juju.is/docs/dev .

**Comment by tmihoc** [comment]
@carkod @anthonydillon Any updates? Thanks!
